src/clarity-angular/layout/_card.clarity.scss

```diff
@@ -120,173 +113,173 @@ $card-children-space-between: 0.5rem !default;
         @include clr-getTypePropertiesForDomElement(card_title, (color, font-size, font-weight, letter-spacing));
     }
 
-    .card-text {
-        font-size: $card-text-fontsize;
-    }
-
-    //Images
-    .card-img {
-        &:first-child > img {
-            border-radius: $clr-default-borderradius $clr-default-borderradius 0 0;
+        .card-text {
```

Comment:
These look like formatting changes. Are they needed?

Reply:
I'll fix these.

Reply:
This was introduced when rebasing. I will fix the indentation.